### ISSUE TITLE
Wrong encoding of unicode characters over SNS

### ISSUE THREAD

**Issue by simonDeHero** [localstack/localstack#1109]
Hi,
my current problem is, that when i publish messages containing unicode characters over SNS, and then retrieve the message via subscribed queue, the payload is wrong (wrong encoding?)

e.g.:
```
aws --endpoint-url=http://localhost:4575 sns create-topic --name my_topic
aws --endpoint-url=http://localhost:4576 sqs create-queue --queue-name my_queue
aws --endpoint-url=http://localhost:4575 sns subscribe --topic-arn arn:aws:sns:eu-west-1:123456789012:my_topic --protocol sqs --notification-endpoint arn:aws:sns:us-east-1:123456789012:my_queue
```

then when just using SQS, unicode payload is ok:
```
aws --endpoint-url=http://localhost:4576 sqs send-message --queue-url http://localhost:4576/queue/my_queue --message-body ö§
aws --endpoint-url=http://localhost:4576 sqs receive-message --queue-url http://localhost:4576/queue/my_queue
// "Body": "ö§", ...
```

but when publishing to SNS, unicode characters are wrong:
```
aws sns publish --topic-arn arn:aws:sns:eu-west-1:123456789012:my_topic --endpoint-url http://localhost:4575 --message ö§
aws --endpoint-url=http://localhost:4576 sqs receive-message --queue-url http://localhost:4576/queue/my_queue
// "Body": "{\"Message\": \"\\u00c3\\u00b6\\u00c2\\u00a7\", ... => Ã¶Â§ (https://www.online-toolz.com/tools/text-unicode-entities-convertor.php)
```

my localstack version is 0.8.10

thx for having a look!
bye
Simon

**Comment by izhelev** [comment]
Having the same issue when publishing to sns. My message contains the string 

> a1'"_!?,. £$- 

 which gets converted to 

>a1'"_!?,. Â£$-

**Comment by whummer** [comment]
Thanks for reporting @simonDeHero @izhelev . This should be fixed in #1182 - can you please give it a try with the latest version on `master` (or `latest` tag of the Docker image).

**Comment by simonDeHero** [comment]
@whummer works! thx!
